code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -47,24 +138,23 @@ var StageProgress = React.createClass({
       }
 
       return ([
-        <div className={outerClass}>
-          <a
-            href={level.url}
-            onClick={onClick}
-            className={innerClass + ' level-' + level.id}
-            style={dotStyle}>
-              {level.title}
-          </a>
-        </div>,
+        <a
+          key={uid}
+          className={`level-${level.id}`}
+          href={level.url}
+          onClick={onClick}
+          style={[dotStyle, style.status[level.status || 'not_tried']]}>
+            {level.title}
+        </a>,
         ' '
       ]);
     });
 
     return (
-      <div className={this.props.largeDots ? 'games' : 'progress_container'}>
+      <div className='react_stage' style={this.props.largeDots ? style.overviewContainer : style.headerContainer}>
```

Comment:
Is overview vs. header the difference between drop-down view and in-header stage view, or something else?